pkg/arvo/app/publish.hoon

```diff
@@ -542,6 +547,63 @@
 ::
 |_  bol=bowl:gall
 ::
+++  get-last-update
+  |=  [host=@p book-name=@tas]
+  ^-  (unit @da)
+  =/  book  (~(get by books) host book-name)
+  ?~  book  ~
+  =/  wen  date-created.u.book
+  %-  some
+  %-  ~(rep by notes.u.book)
+  |=  [[@tas =note] out=_wen]
+  ^-  @da
+  =?  out  (gth last-edit.note wen)
+    last-edit.note
+  =.  out
+    %-  ~(rep by comments.note)
+    |=  [[@da =comment] out=_out]
+    ?:  (gth date-created.comment out)
+      date-created.comment
+    out
+  out
```

Comment:
You could just use `+max` instead of hand-rolling comparisons.

```suggestion
  %+  max  out
  %+  max  last-edit.note
  %-  ~(rep by comments.note)
  |=  [[@da =comment] out=_out]
  (max out date-created.comment)
```